x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -760,3 +760,35 @@ Bamford           |ord
 Bernatsky         |sky
 // end::right-result[]
 ;
+
+endsWithConstant
+from employees | sort emp_no | limit 10 | eval f_a = ends_with(first_name, "a") | keep emp_no, first_name, f_a;
+
+emp_no:integer | first_name:keyword  | f_a:boolean
+10001 | Georgi    | false
+10002 | Bezalel   | false
+10003 | Parto     | false
+10004 | Chirstian | false
+10005 | Kyoichi   | false
+10006 | Anneke    | false
+10007 | Tzvetan   | false
+10008 | Saniya    | true
+10009 | Sumant    | false
+10010 | Duangkaew | false
+;
+
+endsWithSequence
+from employees | sort emp_no | limit 10 | eval f_a = ends_with(first_name, "an") | keep emp_no, first_name, f_a;
```

Comment:
nit: similarly to the tests for `starts_with`, it would be cool to have a test where the arguments for `ends_with` are both fields.

I see that the `employees.csv` does not provide a super obvious way to put together an example. I managed to come up with the following, using the `right` function:

```
endsWithExpression
from employees | sort emp_no | limit 10
| eval last_name_last_letter = right(last_name, 1)
| eval same_last_letters  = ends_with(first_name, last_name_last_letter)
| keep emp_no, first_name, last_name, same_last_letters;

emp_no:integer | first_name:keyword  | last_name:keyword | same_last_letters:boolean
10001 | Georgi    | Facello   | false
10002 | Bezalel   | Simmel    | true
10003 | Parto     | Bamford   | false
10004 | Chirstian | Koblick   | false
10005 | Kyoichi   | Maliniak  | false
10006 | Anneke    | Preusig   | false
10007 | Tzvetan   | Zielinski | false
10008 | Saniya    | Kalloufi  | false
10009 | Sumant    | Peac      | false
10010 | Duangkaew | Piveteau  | false
;
```

This passes fine, btw :)

Reply:
Totally, the more tests the better. Thanks for writing this up, I've included it in the PR.